include/openmc/settings.h

```diff
@@ -53,14 +54,19 @@ extern bool surf_source_write;     //!< write surface source file?
 extern bool surf_mcpl_write;       //!< write surface mcpl file?
 extern bool surf_source_read;      //!< read surface source file?
 extern bool survival_biasing;      //!< use survival biasing?
+extern bool survival_normalization;//!< use survival normalization?
 extern bool temperature_multipole; //!< use multipole data?
 extern "C" bool trigger_on;        //!< tally triggers enabled?
 extern bool trigger_predict;       //!< predict batches for triggers?
 extern bool ufs_on;                //!< uniform fission site method on?
 extern bool urr_ptables_on;        //!< use unresolved resonance prob. tables?
 extern "C" bool weight_windows_on; //!< are weight windows are enabled?
-extern bool write_all_tracks;      //!< write track files for every particle?
-extern bool write_initial_source;  //!< write out initial source file?
+extern bool weight_window_checkpoint_surface;   //!< enable weight window check
```

Comment:
I think these are from another PR - if you rebase on develop these should go away.